code/game/gamemodes/gang/gang.dm

```diff
@@ -69,13 +80,33 @@ GLOBAL_LIST_INIT(gang_colors_pool, list("red","orange","yellow","green","blue","
 	if(gangs.len < 2) //Need at least two gangs
 		return 0
 
-	return 1
+	//turn off sec and captain
+	SSjob.DisableJob(/datum/job/captain)
```

Comment:
I like the idea of just the captain being the Only Sane Man trying to fend off the rabid murderers